rma/wizard/stock_picking_return.py

```diff
@@ -29,6 +29,9 @@ def _onchange_create_rma(self):
                 [("company_id", "=", self.picking_id.company_id.id)]
             ).mapped("rma_loc_id")
             rma_loc_domain = [("id", "child_of", rma_loc.ids)]
+            # We want to avoid setting the return move `to_refund` as it will change
+            # the delivered quantities in the sale and set them to invoice.
+            self.product_return_moves.to_refund = False
```

Comment:
Singleton error if not?

```suggestion
            self.product_return_moves.update({"to_refund": False})
```

Reply:
Not a problem at all as it's a chaing of recordsets. You could even do this with no harm:

```python
env["stock.return.picking"].product_return_moves.to_refund = False
```